Tools/SusyUtility.cc

```diff
@@ -0,0 +1,39 @@
+// SusyUtility.cc
+// Caleb Smith
+// December 13, 2018
+
+// utility functions
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <iterator>
+
+namespace SusyUtility
+{
+    template<typename T>
+    void split(const std::string &s, const char& delim, T result) {
```

Comment:
I'm suspicious of this function.  Please remove the template, because this is not generalizable